### ISSUE TITLE
Jaws read IBM logo/button incorrectly.

### ISSUE THREAD

**Issue by peterdobal** [carbon-design-system/carbon-for-ibm-dotcom#6232]
## Title line template: [Title]: Brief description

Wer using accessibility  tool JAWS reader and  this tool is unable to read  ibm logo button correctly.

## Detailed description

> Describe in detail the issue you're having.

This is accessibility related issue

> Is this a feature request (new component, new icon), a bug, or a general
> issue?
accessibility BUG

> Is this issue related to a specific component?

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?
FF
> What version of Carbon for IBM.com are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

1.Navigate to https://www.ibm.com/docs/en in Firefox
2.Start JAWS
3.Navigate to  left upper IBM logo  button 
4....

Actual  behavior:
Jaws read  on  IBM button/link: "IBM registered" 

Expected behavior:
I think Jaws should read "IBM"

## Additional information

Same behavior on ibm.com

- Screenshots or code
- Notes

**Comment by DragosRistici** [comment]
When the link is focused, "link, IBM registered" is read out. I'm not sure that was not intended. @Praveen-sr can investigate further. 

<img width="260" alt="Screenshot 2021-06-02 at 14 25 59" src="https://user-images.githubusercontent.com/48327647/120472466-9830e880-c3ae-11eb-948f-b5e7ac4f084c.png">

**Comment by DragosRistici** [comment]
**As per Praveen check:** 
Just checked the link, "IBM registered" is fine...its not a bug. Best practice would be "IBM logo link"
JAWS reads out its graphic link so the user will know its logo link.

Thank you, @peterdobal for this one. We'll close this and when time permits, reopen it in the near future.
